src/templates/layouts/healthCareLocalFacility/index.tsx

```diff
@@ -22,13 +24,53 @@ export function HealthCareLocalFacility({
   path,
   administration,
   vamcEhrSystem,
+  officeHours,
+  address,
+  phoneNumber,
+  image,
+  facilityLocatorApiId,
+  geoLocation,
 }: FormattedHealthCareLocalFacility) {
   // Populate the side nav data for the side nav widget to fill in
   // Note: The side nav widget is in a separate app in the static-pages bundle
   useEffect(() => {
     window.sideNav = menu
   })
 
+  const structuredSchemaData = {
+    '@context': 'https://schema.org',
+    '@type': 'Place',
+    address: {
+      '@type': 'PostalAddress',
+      streetAddress: `${address.address_line1}${
+        address.address_line2 ? `, ${address.address_line2}` : ''
+      }`,
+      addressLocality: address.locality,
+      addressRegion: address.administrative_area,
+      postalCode: address.postal_code,
+    },
+    name: title,
+    telephone: phoneNumber,
+    openingHoursSpecification: officeHours.map((hours) => ({
+      '@type': 'OpeningHoursSpecification',
+      dayOfWeek: `https://schema.org/${dayOfWeek(hours.day)}`,
+      opens: numberToTimeString(hours.starthours),
+      closes: numberToTimeString(hours.endhours),
```

Comment:
`Event` will probably need to do this as well so it matches the schema defined for `'@type': 'Place'`.